test/solidity-ibc/IBCAdminTest.t.sol

```diff
@@ -156,6 +162,58 @@ contract IBCAdminTest is Test {
         ics26Router.setTimelockedAdmin(newTimelockedAdmin);
     }
 
+    function test_success_setPortCustomizer() public {
```

Comment:
I just want to make sure, because I don't remember, but do we have tests that cover that someone without access cannot use this without the role?

Reply:
Added it now